src/SSHLibrary/library.py

```diff
@@ -394,12 +407,12 @@ def set_default_configuration(self, timeout=None, newline=None, prompt=None,
         [#Default encoding|`encoding`]
         were added in SSHLibrary 2.0.
         """
-        self._config.update(timeout=timeout, newline=newline, prompt=prompt,
+        self._config.update(timeout=timeout, newline=newline, prompt=prompt, prompt_is_regexp=prompt_is_regexp,
```

Comment:
Not completely sure what is the policy, but at a glance looks like rest of code tries to stay within 80 char limit. Some additions break this.

Reply:
I was still in the 80 char limit, but you're right the code looks messy and weird,. I will make it in line with the rest of the code.